src/test/specs/ExpectNoAttributeValueSpec.ts

```diff
@@ -18,4 +18,14 @@ describeCommon('expectNoAttributeValue', () => {
   it('Exact match error thrown', () => {
     expect(() => BrowserUtils.expectNoAttributeValue(EMPTY_DIV, 'align', 'center')).to.throw(Error);
   });
+
+  it('Case Element not found.', () => {
+    BrowserUtils.navigateToUrl(sampleAppUrl);
+    expect(() => BrowserUtils.expectNoAttributeValue('//NotExist', 'method', 'post')).to.throw(Error);
+  });
+
+  it('Case attribute not found', () => {
```

Comment:
suggestion for name : "Not existing attribute"

Reply:
changed to : "Attribute not exists"